src/ui/settings.ts

```diff
@@ -238,6 +241,54 @@ export class SettingTab extends PluginSettingTab {
     });
   }
 
+  generateCustomRegexReplacementSettings(tabName: string, containerEl: HTMLElement): void {
```

Comment:
This is mostly good to go. It just looks like it is missing in the list of rules to search which requires a container el that is not the direct container el for another setting, the tab name, and a set of information to search on .